drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -4463,8 +4463,9 @@ void RasterizerStorageGLES3::mesh_render_blend_shapes(Surface *s, const float *p
 	for (int ti = 0; ti < mtc; ti++) {
 		float weight = p_weights[ti];
 
-		if (weight < 0.00001) //not bother with this one
+		if (weight <= 0.00001 && weight >= -0.00001) {
```

Comment:
Just a minor thing, in this way it's more clear:
`if (weight >= -0.00001 && weight <= 0.00001) {`